helpers/test-helper.js

```diff
@@ -329,7 +330,7 @@ export async function getValidEntryNumber(colonyNetwork, account, hash, starting
   const userBalance = userLockInformation.amount;
 
   // What's the largest entry they can submit?
-  const nIter = userBalance.div(new BN(10).pow(new BN(15)));
+  const nIter = userBalance.div(new BN(10).pow(new BN(18)).muln(2000));
```

Comment:
Sorry just having a last look through this.
Isn't the divisor here `MIN_STAKE`?

Reply:
Yes, but because `constants.js` imports from `test-helper.js`, you can't import `MIN_STAKE` because of circularity.